staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -47,22 +55,36 @@ func (o *AuditLogOptions) AddFlags(fs *pflag.FlagSet) {
 		"The maximum number of old audit log files to retain.")
 	fs.IntVar(&o.MaxSize, "audit-log-maxsize", o.MaxSize,
 		"The maximum size in megabytes of the audit log file before it gets rotated.")
+
+	fs.StringVar(&o.WebhookConfigFile, "audit-log-webhook-config-file", o.WebhookConfigFile,
+		"File with webhook configuration for audit events in kubeconfig format.")
 }
 
 func (o *AuditLogOptions) ApplyTo(c *server.Config) error {
-	if len(o.Path) == 0 {
-		return nil
+	var backends []audit.Backend
```

Comment:
@timstclair looks like there's some overlap with https://github.com/kubernetes/kubernetes/pull/46009. Inclined to let your PR merge first since that actually hooks up the feature flag.